client/blocks/inline-help/inline-help-search-results.scss

```diff
@@ -0,0 +1,91 @@
+.inline-help__empty-results {
+	padding: 8px 16px 0;
+	margin: 0;
+	font-size: $font-body-small;
+	color: var( --color-text-subtle );
+	font-style: italic;
+	text-align: left;
+
+	@include breakpoint-deprecated( '>660px' ) {
+		font-size: $font-body;
+	}
+}
+
+.inline-help__results {
+	padding: 8px 0;
+}
+
+.inline-help__results-list {
+	list-style: none;
+	text-align: left;
+	margin: 0;
+	padding: 0;
+}
+
+.inline-help__results-title {
+	font-weight: bold;
+	line-height: 1.4;
+	display: block;
+	padding: 8px 16px;
+	font-size: $font-body-small;
+	text-align: left;
+
+	&::after {
+		content: ':';
+	}
+}
+
+.inline-help__results-cell {
+	width: 100%;
+}
+
+.inline-help__results-item {
+	margin: 0;
+	font-size: $font-body-small;
+
+	@include breakpoint-deprecated( '>660px' ) {
+		font-size: $font-title-small;
+		line-height: 1.3;
+	}
+
+	a {
+		//color: var( --color-primary );
+		text-decoration: underline;
+		font-weight: normal;
+		line-height: 1.4;
+		display: flex;
+		padding: 8px 16px;
+
+		span {
```

Comment:
It's a bummer we're using spans and not a classnames for the title wrapper. Pre-existing, so feel free to ignore.